docs/collections/_other/doc-history.md

```diff
@@ -7,28 +7,29 @@ nav_order: 14
 | Added [Schema grammar](../schema/schema-grammar.html) topic | July 17, 2023 |
 | Added [JSON policy format](../policies/json-format.html) reference page | July 14, 2023 |
 | Added [Best Practices](../overview/best-practices.html) topics | July 14, 2023 |
-| Initial release of the Cedar Policy Language Guide | May 10, 2023 | 
-
+| Initial release of the Cedar Policy Language Guide | May 10, 2023 |
 
 ## History of the Cedar language
 
-| Cedar<br/>Version | Description | Cedar SDK<br/>Versions | Date | 
-| --- |--- |--- | --- |
-| 3.0.0 | New `is` operator [PR](https://github.com/cedar-policy/cedar/pull/396) | 3.0.0 | December 15, 2023 |
-| 2.1.2 | Change to how validation treats template-linked policies [PR](https://github.com/cedar-policy/cedar/pull/371) | 2.4.2 | October 23, 2023 |
-| 2.1.1 | Increased validation precision [PR](https://github.com/cedar-policy/cedar/pull/117) | 2.4.0 - 2.4.1 | September 29, 2023 |
-| 2.1.0 | [RFC #9](https://github.com/cedar-policy/rfcs/blob/main/text/0009-disallow-whitespace-in-entityuid.md): Disallows whitespace in namespaces | 2.3.0 - 2.3.3 | June 29, 2023 |
-| 2.0.0 | Initial release of the Cedar Policy Language | 2.0.0 - 2.2.0 | May 10, 2023 |
-
-
-
+The following table tracks changes to the Cedar language version. The language version is different from the SDK version (which corresponds to the `cedar-policy` version on [crates.io](https://crates.io/crates/cedar-policy)) because a breaking change for the Cedar Rust API may or may not be a breaking change for the Cedar language. See the [SDK changelog](https://github.com/cedar-policy/cedar/blob/main/cedar-policy/CHANGELOG.md) for details about what is included in different SDK versions. We recommend using the latest SDK version that supports your desired language version since later SDK versions may contain important bug fixes and improvements to usability.
+
+| Cedar<br/>Version | Description | Cedar SDK<br/>Version(s) | Date |
+| --- | --- | --- | --- |
+| 3.3 | References between common types ([issue#154](https://github.com/cedar-policy/cedar/issues/154)) | 3.2.0 - 3.2.1 | May 17, 2024 |
+| 3.2 | General multiplication operator ([rfc#57](https://github.com/cedar-policy/rfcs/blob/main/text/0057-general-multiplication.md)) | 3.1.2 - 3.1.4 | March 29, 2024 |
+| 3.1 | New schema syntax ([rfc#24](https://github.com/cedar-policy/rfcs/blob/main/text/0024-schema-syntax.md)) | 3.1.0 - 3.1.1 | March 8, 2024 |
+| 3.0 | `is` operator ([rfc#5](https://github.com/cedar-policy/rfcs/blob/main/text/0005-is-operator.md))<br/>Stricter validation ([rfc#19](https://github.com/cedar-policy/rfcs/blob/main/text/0019-stricter-validation.md))<br/>Disallow duplicate keys in records ([rfc#20](https://github.com/cedar-policy/rfcs/blob/main/text/0020-unique-record-keys.md))<br/>Request validation ([issue#191](https://github.com/cedar-policy/cedar/issues/191)) | 3.0.0 - 3.0.1 | December 15, 2023 |
```

Comment:
If we include request validation, should we also include entity data validation?

Reply:
Makes sense. Added in the latest commit.